src/app.js

```diff
@@ -0,0 +1,102 @@
+const express =require( "express");
+const dotenv =require("dotenv") ;
+const cors =require( "cors");
+const helmet =require("helmet") ;
+const xss =require("xss-clean") ;
+const rateLimit = require("express-rate-limit") ;
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9973834)
```suggestion
const rateLimit = require("express-rate-limit");
```